schema/bom-1.5.schema.json

```diff
@@ -42,6 +42,7 @@
       "type": "integer",
       "title": "BOM Version",
       "description": "Whenever an existing BOM is modified, either manually or through automated processes, the version of the BOM SHOULD be incremented by 1. When a system is presented with multiple BOMs with identical serial numbers, the system SHOULD use the most recent version of the BOM. The default version is '1'.",
+      "minimum": 1,
```

Comment:
:warning: attention: 
here i changed the type from integer to positiveInteger.
this was done, so that the pattern for BOM-Link could be crafted properly.
If this was an integer, rather than a positiveInteger, then the pattern would need to allow signs `-/+`